ONSdigital/eq-author
# Add move section feature

### What is the context of this PR?
- Implements functionality required to re-order and move sections within a questionnaire

### How to review 
- Run tests 
- Ensure feature works as per https://trello.com/c/TAzBTkXp/454-re-ordering-sections-8 

### Dependencies
- API: https://github.com/ONSdigital/eq-author-api/pull/94

<img width="1277" alt="screen shot 2018-07-27 at 12 47 09" src="https://user-images.githubusercontent.com/2471296/43319175-a532a9da-919b-11e8-8681-4f9caac8be3d.png">
